rplugin/python3/deoplete/sources/deoplete_clang.py

```diff
@@ -265,7 +265,7 @@ def get_compilation_database(self, fname):
                              os.path.realpath(os.path.join(cwd, arg))]:
                         skip = 0
                         continue
-                    elif arg == '-o':
+                    elif arg == '-o' or arg[0] == '/':
```

Comment:
Actually this is not the right fix.

The first if line 263 should remove the file name after the '-c', I don't understand why it did'nt work in my case.

How can I debug this? For now raised Exception that was catch by deoplete, but is there an other way?